boards/xg23b-pk6068a/include/periph_conf.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2022 SSV Software Systems GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_xg23b-pk6068a
+ * @{
+ *
+ * @file
+ * @brief       Configuration of CPU peripherals for xG23B-PK6068A board
+ *
+ * @author      Juergen Fitschen <me@jue.yt>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include <stdint.h>
+
+#include "cpu.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Clock configuration
+ * @{
+ */
+#define HFXO_FREQ           (39000000UL)
```

Comment:
I couldn't find any information in the documentation or the schematic what the oscillator frequency for HF on the Radio board is. I could only find the 26 MHz oscillator on Main Board.

Reply:
This is the schematic of the radio board: https://www.silabs.com/documents/public/schematic-files/BRD4210A-A01-schematic.pdf

Reply:
Yes, I saw exactly this schematic, but I only found the LFXO there. I simply missed the RF oscillator in the radio interface on the second page :sunglasses: